nntrainer/tensor/var_grad.h

```diff
@@ -211,6 +211,20 @@ class Var_Grad {
    */
   Tensor &getGradientRef() { return *grad.get(); }
 
+  /**
+   * @brief Get the variable tensor (by reference)
+   *
+   * @return Tensor Variable tensor
+   */
+  Tensor &getVariableRef() const { return *var.get(); }
```

Comment:
return const reference? (and also for :226)

Reply:
In many cases, we need to modify the tensor or call in_place operations on this.
That won't be possible with const.

Reply:
If modifying tensor take places, shouldn't it be `var_grad` rather than `const var_grad` in the first place?

Reply:
done.

Reply:
Glad that it didn't.